src/schema/type/Annotation.graphql

```diff
@@ -0,0 +1,98 @@
+    field: String!
+    "In the case that the target is a fragment, the fragment value"
+    fragment: String
+}
+
+interface AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+# TODO: This will require creating an object of this type for each item in the enum AnnotationMotivation
+type AnnotationSchemaMotivation implements AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    motivation: AnnotationMotivation!
+}
+
+"A custom motivation"
+type AnnotationCEMotivation implements AnnotationMotivationInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    # TODO: This should be rendered in jsonld as skos:Broader
+    parentMotivation: AnnotationMotivation!
+    motivation: String!
+}
+
+interface AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+}
+
+"http://www.w3.org/ns/oa#TextualBody"
+type AnnotationTextualBody implements AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#value"
+    value: String!
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    # Should always be "http://www.w3.org/ns/oa#TextualBody"
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+    type: String
+    # TODO: Purpose (same as a motivation, can it be a custom value?)
+}
+
+"An annotation body which refers to a node in the CE"
+type AnnotationCEBody implements AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    target: Thing!
+}
+
+"An annotation body which refers to an external URL"
+type AnnotationURLBody implements AnnotationBodyInterface {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    target: URL!
+}
+
+"http://www.w3.org/ns/oa#Annotation"
+type Annotation {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://www.w3.org/ns/oa#hasTarget"
+    target: [AnnotationTargetInterface] @relation(name: "ANNOTATION_TARGET", direction: "OUT")
+    "http://purl.org/dc/terms/creator"
+    creator: URL
+    "http://purl.org/dc/elements/1.1/date"
+    created: String
+    "http://purl.org/dc/elements/1.1/date"
+    updated: String
+    "http://www.w3.org/ns/oa#Motivation"
+    motivation: [AnnotationMotivationInterface] @relation(name: "ANNOTATION_MOTIVATION", direction: "OUT")
+    "http://www.w3.org/ns/oa#hasBody"
+    body: [AnnotationBodyInterface] @relation(name: "ANNOTATION_BODY", direction: "OUT")
+    # TODO: Audience
```

Comment:
@musicog to confirm if Audience is wanted

Reply:
Again this is a use-case question; however, this could be rather useful in scoping performative annotations to a particular part or parts (orchestra use case). I can think of other potential use-cases as well (e.g., certain Mahler-festival-annotation being displayed only to audiences that are able to read notation, etc), but it's really down to the use case.

Reply:
After a discussion we decided that it would be nice to be able to refer to each of our 5 user types (scholars, instrumental players, choir singers, orchestras, music enthusiasts) as an audience.
We will render this by defining in our vocabulary (https://github.com/trompamusic/ce-api/issues/87) an Audience for each of these types, based on https://schema.org/PeopleAudience. If in the future we want to add more specific audiences (e.g. "string players", "mahler scholars") then we can decide how to model more specific audience types.